apps/charterafrica/package.json

```diff
@@ -51,9 +51,12 @@
     "@nivo/pie": "^0.84.0",
     "@nivo/tooltip": "^0.84.0",
     "@nivo/waffle": "^0.84.0",
+    "@payloadcms/bundler-webpack": "^1.0.5",
+    "@payloadcms/db-mongodb": "^1.2.0",
```

Comment:
I dont think it's a showstopper, but we could maybe use the latest `@payloadcms/db-mongodb@1.32` package. The latest bug fixes have some issues affecting transactions